### DISCUSSION TITLE
求助，vite打包库模式怎么设置中文

### DISCUSSION THREAD

**Discussion by tetezi** [element-plus/element-plus#16637] [Q&A]
求助，在写一个自己的二次封装的组件库，但是发现导出的组件里的element组件都是英文的，该怎么解决啊，
![图片](https://github.com/element-plus/element-plus/assets/76197961/f5f78761-8e15-43e8-8ce3-11146b142fb3)
我的打包入口文件这里实际上也不会涉及到dom和app什么的，毫无头绪，难道真的要每个组件都去包一层el-config-provider吗。
![QQ图片20240408181505](https://github.com/element-plus/element-plus/assets/76197961/33f544ef-54a0-42ed-a9ec-5300f1ac0318)

**Comment by xxholly32** [comment]
业务上使用的时候包一层el-config-provider就好了
